targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/device/TOOLCHAIN_GCC_ARM/MK64FN1M0xxx12.ld

```diff
@@ -53,9 +53,8 @@ ENTRY(Reset_Handler)
 
 __ram_vector_table__ = 1;
 
-/* Heap 1/4 of ram and stack 1/8 */
-__stack_size__ = 0x8000;
-__heap_size__ = 0x10000;
+__stack_size__ = 0x400;
```

Comment:
@Patater In both 3 files, it's probably a good idea to add a short comment stating that:

* These values do not affect the actually available heap for an app (they only guarantee a min).
* They do not affect the `main` stack size.
* They are set *that way* because of uVisor.